examples/dashboard_basic_crud.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+#
+# Simple example of dashboard creation, retrieval, updating, and deletion.
+#
+
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), '..'))
+from sdcclient import SdMonitorClient
+import uuid
+
+#
+# Parse arguments
+#
+if len(sys.argv) != 2:
+    print('usage: %s <sysdig-token>' % sys.argv[0])
+    print('You can find your token at https://app.sysdigcloud.com/#/settings/user')
+    sys.exit(1)
+
+sdc_token = sys.argv[1]
+
+#
+# Instantiate the SDC client
+#
+sdclient = SdMonitorClient(sdc_token)
+
+#
+# Create Dashboard.
+#
+ok, res = sdclient.create_dashboard("Sample dashboard - " + uuid.uuid4().hex)
+
+#
+# Check for successful creation
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Get Dashboard.
+#
+ok, res = sdclient.get_dashboard(dashboard['id'])
+
+#
+# Check for successful retrieval
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Update Dashboard.
+#
+dashboard['name'] = "Let's change the dashboard name. " + uuid.uuid4().hex
+ok, res = sdclient.update_dashboard(dashboard)
+
+#
+# Check for successful update
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Delete Dashboard.
+#
+ok, res = sdclient.delete_dashboard(dashboard)
+
+#
+# Check for successful delete
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+#
+# Create Dashboard.
```

Comment:
This works now!

Minor comments:
1. Since this is a different use case shown, I'd suggest to add a comment to indicate that here you are showing how one could overwrite an existing dashboard with a configuration of a different one
2. I'd remove (or at least deemphasize) comments for the duplicated parts (eg. check results, delete dashboard) since they may be a little distracting (I'd focus the attention on what's different from the previous use case)
3. `dashboard['name'] = new_dashboard['name']` doesn't seem to meet the goal I initially understood: If the goal is to overwrite a dashboard, I'd probably find it more intuitive to retain the "new" dashboard name, not the "existing" one